### DISCUSSION TITLE
Theme working on localhost but not when turned into a static site

### DISCUSSION THREAD

**Discussion by drschatz** [thegeeklab/hugo-geekdoc#812] [Q&A] [answered]
Hello,
I really want to use this theme and it looks great when I use it locally with `hugo server` command on a local host. However, when I run `hugo` the index.html produced in the public folder does not look good in my browser. It looks like all the CSS isn't found and it's not linked correctly to other pages. Any ideas on what I'm doing wrong? Thank you!
<img width="1473" alt="Screenshot 2024-04-05 at 6 08 38 AM" src="https://github.com/thegeeklab/hugo-geekdoc/assets/142695154/c8e031fe-19f7-4edd-aa08-68ec731ae621">

**Comment by xoxys** [comment]
Hi, this is normal and corresponds to the way websites work. You cannot simply load the index.html in your browser, but must use a web server to deliver the page. The `hugo server` command does this under the hood and starts a local web server for you at http://localhost:1313. As this is a dev server, it should not be used in production.

**Reply by drschatz** [reply]
I guess I'm confused, since I thought this was a static website. To clarify, If I host these files, say on github's web services, would it work?

**Reply by xoxys** [reply, marked as answer]
> I guess I'm confused, since I thought this was a static website. 

Static means, no dynamic language like PHP is involved. However, plain html/css/js websites still need to web server to work.

> To clarify, If I host these files, say on github's web services, would it work?

Yes, if you publish the build output from the `public` directory to a GitHub page branch, for example, and set up GH pages, GH will basically manage a web server for you to serve the page.
